lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -255,11 +255,20 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
 
         if (isset($xmlRoot->embedded)) {
             foreach ($xmlRoot->embedded as $embeddedMapping) {
+                $columnPrefix = isset($embeddedMapping['column-prefix'])
```

Comment:
Actually, this fix is incorrect, now that I look at it. See description of issue http://www.doctrine-project.org/jira/browse/DDC-3293 - I used an additional property there, and that is because `true` and `false` are still valid string prefixes.